TradeItIosTicketSDK2/SerializedLinkedBroker.h

```diff
@@ -0,0 +1,21 @@
+//
+//  SerializedLinkedBroker.h
+//  TradeItIosTicketSDK2
+//
+//  Created by Guillaume DEBAVELAERE on 17/08/2017.
+//  Copyright © 2017 TradeIt. All rights reserved.
+//
+
+#import <JSONModel/JSONModel.h>
+#import "SerializedLinkedBroker.h"
+#import "SerializedLinkedBrokerAccount.h"
+
+@interface SerializedLinkedBroker : JSONModel
+
+@property (nonatomic) NSArray<SerializedLinkedBrokerAccount*> <SerializedLinkedBrokerAccount> *accounts;
```

Comment:
This may not be the case: but could we use any of the existing JSONModels we have instead of creating new models to serialize?